spec/lib/fake_data/complaints_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+RSpec.describe FakeData::Complaints do
+  describe "complaint_data" do
+    it "returns a single complaint object" do
+      complaint = FakeData::Complaints.data
+
+      expect(complaint[:id]).to_not be_nil
+      expect(complaint[:attributes]).to be_a(Hash)
+      expect(complaint.dig(:relationships, :grantAward, :meta, :id)).to be_a(String)
+      expect(complaint[:links]).to be_a(Hash)
+    end
+  end
+
+  describe "datetime" do
+    it "returns a string formatted as ISO-8601 UTC datetime" do
+      dt = FakeData::Complaints.datetime
+      expect(dt).to be_a(String)
+      begin
+        expect(Time.iso8601(dt)).to be_a(Time)
+      rescue
+        expect(false).to be_truthy
+      end
+    end
+  end
+
+  describe "identifier" do
+    it "returns a string of numbers" do
+      id = FakeData::Complaints.identifier
+      expect(id).to be_a(String)
+      expect(id.to_i.to_s).to eq(id)
```

Comment:
i'd prefer this to be 

`expect(id.to_i).not_to eq 0` (assuming this HAS to be a string of numbers)

Reply:
It doesn't have to be but it currently is just to populate it with something

Reply:
I don't think that `expect(id.to_i).not_to eq 0` works entirely, because `"12".to_i` returns 12, but also it doesn't really matter what the value of this is.  Maybe I'll just remove the test checking the substance and be happy that there is an identifier which is a string

Reply:
+1 to not worrying about the substance of this identifier.